administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue

```diff
@@ -97,6 +98,9 @@
             },
             isGridSize(size) {
                 return (this.$store.state.gridSize === size);
+            },
+            changeSearch(query){
+                this.$store.commit(types.SEARCH_QUERY, query.target.value);
```

Comment:
Please create a mutation for the commit, you have not created a mutation

Reply:
Added

Reply:
You can use more meaningful commit name like `SET_SEARCH_QUERY` so the name will be self describing